dtests/src/test/java/io/snappydata/hydra/testDMLOps/SnappyDMLOpsUtil.java

```diff
@@ -1185,6 +1186,11 @@ public String getRowFromCSV(String tableName, int randTable) {
     //    File.separator + csvFileName);
     try (Stream<String> lines = Files.lines(Paths.get(csvFilePath + File.separator + csvFileName))) {
       row = lines.skip(insertCounter).findFirst().get();
+    } catch (NoSuchElementException nse) {
```

Comment:
Since it not required in all tests, this can be made flag based, we have a flag with the bug number, why we need to re-insert the data.

Reply:
Done.